src/ui/models/conv.coffee

```diff
@@ -24,7 +24,7 @@ add = (conv) ->
     # participant_data contains entity information
     # we want in the entity lookup
     entity.add p for p in conv?.participant_data ? []
-    later -> action 'syncrecentconversations'
+    later -> action 'syncrecentconversationsnow'
```

Comment:
The call to syncrecentconversationsnow is causes an issue when sending a message where the text disappears for a short time between it's sending state and when it shows as black  text that has been sent.